thoth-api/src/model/biography/crud.rs

```diff
@@ -36,15 +38,21 @@ impl Crud for Biography {
         filter: Option<String>,
         order: Self::OrderByEntity,
         _: Vec<Uuid>,
-        parent_id_1: Option<Uuid>,
+        _parent_id_1: Option<Uuid>,
```

Comment:
It would make more sense for the unused argument to be `parent_id_2` rather than `parent_id_1` (and whichever is unused can just be named `_`)